src/docs/updateProduct.ts

```diff
@@ -0,0 +1,190 @@
+/**
+ * @swagger
+ * /collections/{id}/product:
+ *   put:
+ *     summary: Update a product
+ *     description: Updates an existing product by ID.
+ *     tags:
+ *       - PRODUCT
```

Comment:
Put the tag in square brackets so that it doesn't isolate from other product endpoints
```suggestion
 *       - [PRODUCT]
```